src/spacegame3/SpaceGame.java

```diff
@@ -19,6 +19,7 @@
 public final class SpaceGame extends Application {
     private static final Logger LOG = Logger.getLogger(SpaceGame.class.getName());
 
+
```

Comment:
removes this line, it adds nothing to the file and may bring conflicts with other branches